static/images/products/jetpack/product-backup.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 64 64">
```

Comment:
Can we minify the svg?

Reply:
I've run it through SVGOMG but I can see that the whitespace has not been removed. I wonder if removing the whitespace is worth it anyway.

Reply:
Well, we usually remove the whitespace, as it's really unnecessary burden for the client to download, and we don't currently minify the svgs out of the box.